pay-api/src/pay_api/models/non_sufficient_funds.py

```diff
@@ -55,15 +57,16 @@ class NonSufficientFundsSchema:  # pylint: disable=too-few-public-methods
     """Used to search for NSF records."""
 
     id: int
-    description: str
+    cfs_account: int
     invoice_id: int
```

Comment:
Still need invoice id?

Reply:
I think it's a bit confusing to leave it in there, because the NSF pertains to multiple invoice ids